DEV - Bugfix: HF Dataset Not Being Updated In Prod

### Overview

We need to ensure the github workflow that updates the HF dataset is setting up its environment variables correctly. Currently, the action is failing due to missing VITE_ENV key.


### Action Items

- [x] Add the following line to the `Setup Environment` job (see doc page in Resources for syntax)
- [x] 202508011: investigate new error in Github Actions
  - see resources: Github Actions Error
- [ ] Leave a comment detailing the cause of the problem
- [ ] Fix and open PR for review

### Resources/Instructions

#### Missing the echo of VITE_ENV...

```bash
echo "VITE_ENV=${{ secrets.VITE_ENV }}" > .env
```

#### Click to see Snippet...

<details><summary>Github Actions Error</summary>
<p>

```
...
14311379it [01:12, 198596.87it/s]
/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/getpass.py:91: GetPassWarning: Can not control echo on the terminal.
  passwd = fallback_getpass(prompt, stream)
Warning: Password input may be echoed.

    _|    _|  _|    _|    _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|_|_|_|    _|_|      _|_|_|  _|_|_|_|
    _|    _|  _|    _|  _|        _|          _|    _|_|    _|  _|            _|        _|    _|  _|        _|
    _|_|_|_|  _|    _|  _|  _|_|  _|  _|_|    _|    _|  _|  _|  _|  _|_|      _|_|_|    _|_|_|_|  _|        _|_|_|
    _|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
    _|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
    
    To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .
Token: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/getpass.py", line 69, in unix_getpass
    old = termios.tcgetattr(fd)     # a copy to save
          ^^^^^^^^^^^^^^^^^^^^^
termios.error: (25, 'Inappropriate ioctl for device')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/311-data/311-data/scripts/updateHfDataset.py", line 104, in <module>
    main()
  File "/home/runner/work/311-data/311-data/scripts/updateHfDataset.py", line 100, in main
    hfUpload()
  File "/home/runner/work/311-data/311-data/scripts/updateHfDataset.py", line 80, in hfUpload
    login(TOKEN)
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/huggingface_hub/_login.py", line 100, in login
    interpreter_login(new_session=new_session, write_permission=write_permission)
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/huggingface_hub/_login.py", line 161, in interpreter_login
    token = getpass("Token: ")
            ^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/getpass.py", line 91, in unix_getpass
    passwd = fallback_getpass(prompt, stream)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/getpass.py", line 126, in fallback_getpass
    return _raw_input(prompt, stream)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/getpass.py", line 148, in _raw_input
    raise EOFError
EOFError
Error: Process completed with exit code 1.
```

</p>
</details> 

#### Other Useful Docs...
- 🌐 https://serverfault.com/questions/196734/bash-difference-between-and-operator
- 🌐 [Github Docs: Workflow Commands For Github Actions, Example of Multiline Markdown Content](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#example-of-multiline-markdown-content)
- 🌐 [Single Quotes vs Double Quotes in YAML: How They Differ From JavaScript and TypeScript](https://dev.to/darkmavis1980/single-quotes-vs-double-quotes-in-yaml-how-they-differ-from-javascript-and-typescript-20hj)
- 🌐 [Why am I merging "remote-tracking branch 'origin/develop' into develop"?](https://stackoverflow.com/questions/6406762/why-am-i-merging-remote-tracking-branch-origin-develop-into-develop)

--- a/.github/workflows/hfUpdate.yml
+++ b/.github/workflows/hfUpdate.yml
@@ -2,7 +2,7 @@ name: Update Huggingface Dataset
 
 on:
   schedule: # once daily at 3:05 pm PST (23:05 UTC)
-    - cron: '5 23 * * *'
+    - cron: "5 23 * * *"
 
 jobs:
   run:
@@ -11,20 +11,20 @@ jobs:
       run:
         working-directory: scripts/
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
+      - name: Checkout code
+        uses: actions/checkout@v2
 
-    - name: Set up Python environment
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.11.x'
+      - name: Set up Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.11.x"
 
-    - name: Install dependencies
-      run: pip install -r requirements.txt
+      - name: Install dependencies
+        run: pip install -r requirements.txt
 
-    - name: Setup environment
-      run: |
-        echo "HUGGINGFACE_LOGIN_TOKEN=${{ secrets.HUGGINGFACE_LOGIN_TOKEN }}" > .env
-
-    - name: Run script
-      run: python updateHfDataset.py
+      - name: Setup environment
+        run: |
+          echo "HUGGINGFACE_LOGIN_TOKEN=${{ secrets.HUGGINGFACE_LOGIN_TOKEN }}" > .env
+          echo "VITE_ENV=${{ secrets.VITE_ENV }}" > .env
+      - name: Run script
+        run: python updateHfDataset.py

